modules/kubernetes/aks-core/modules.tf

```diff
@@ -372,3 +373,15 @@ module "xenit" {
   loki_api_fqdn        = var.xenit_config.loki_api
 }
 
+module "goldpinger" {
+  depends_on = [module.opa_gatekeeper, module.linkerd]
```

Comment:
Do we need linkerd to run goldpinger? Can't we use this on all our clusters for network monitoring?

Reply:
Is terrafrom smart enough that if linkerd module is disabled it will still continue?

Reply:
Yes terraform will be smart enough to ignore this if linkerd is not enabled.